### DISCUSSION TITLE
Ports E and F are prepared for "high-speed" - what does that mean?

### DISCUSSION THREAD

**Discussion by JuliaFiMSFT** [pybricks/support#1260] [Q&A]
In the LEGO Education SPIKE Prime Technical Specifications it says:

"115kB port speed (ports E and F are prepared for "high-speed")"

What does that mean? What can I do to take advantage of the "faster" ports?

https://assets.education.lego.com/v3/assets/blt293eea581807678a/bltf512a371e82f6420/5f8801baf4f4cf0fa39d2feb/techspecs_techniclargehub.pdf?locale=en-us

**Comment by laurensvalk** [comment]
I think they might have hardware control flow pins internally. For the existing sensors/motors, this shouldn't make a difference in terms of speed.

The sensors and motors have a limited data update frequency (100 or 200 Hz depending on device) which is far lower than the UART speed bottleneck.
